packages/database/schema.prisma

```diff
@@ -163,6 +165,7 @@ model Display {
   person    Person?       @relation(fields: [personId], references: [id], onDelete: Cascade)
   personId  String?
   status    DisplayStatus @default(seen)
+  Response  Response[]
```

Comment:
I would like to change this. I think it's the better approach to reference a response (with responseId) in the Display. That way we can stick with the current approach of updating the display instead of needing to reference the display in the response. We could potentially also eliminate the status as we can check if a responseId is set. But we need the status for now for backward compatibility with older versions of formbricks-js.